changelog.md

```diff
@@ -0,0 +1,6 @@
+# Changelog
+
+## [Unreleased]
+
+### Added
+- ON-686: project set up and functions for generating a Form URL with a JWT
```

Comment:
For a `changelog.md` before any releases. This prob should not have anything other than:

## Unreleased

### Initial Release